apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1707,4 +1707,131 @@ describe('[Rooms]', function () {
 				.end(done);
 		});
 	});
+
+	describe('/rooms.muteUser', () => {
+		let testChannel;
+
+		before('create a channel', async () => {
+			const result = await createRoom({ type: 'c', name: `channel.test.${Date.now()}-${Math.random()}` });
+			testChannel = result.body.channel;
+		});
+
+		it('should invite rocket.cat user to room', async () => {
```

Comment:
those tests don't have to be async, a promise is returned from the .expect chain already

Reply:
Nice catch!